### PR TITLE
chore: improve docs after code fence feature

### PR DESCRIPTION
**Pull request by Jonathansoufer**
## **Related issues**

Fixes: #[160](https://github.com/MetaMask/accounts-planning/issues/160)

## **Manual testing steps**

1. Go to the main README and search for code fence information.

## **Screenshots/Recordings**

N/A

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +113,13 @@ cd metamask-mobile
 - _Non-MetaMask Only:_ Fill `MM_SENTRY_DSN` in `.js.env` if you want the app to emit logs to your own Sentry project.
 - Note: after updating env vars, make sure to run `source .js.env` on the root of the project
 
+-   Choose your Build Type
+Metamask has currently two build types: `main` and `flask`. The `main` build type is the _default_ one and is used for production builds. The `flask` build type is used for a canary release, mainly used for development and testing of new/beta features purposes. To change the build type, set the `METAMASK_BUILD_TYPE` environment variable to either `main` or `flask`. For example:
```
Comment (sethkfman) [inline README.md:117]: What about adding this value in the `js.env`?

Comment (Jonathansoufer) [reply README.md:117]: This value is already added into js.env (https://github.com/MetaMask/metamask-mobile/blob/de2c1489e41e515dfe8f3ee77d3ce40dfd5e7c65/.js.env.example#L54)

**Review by owencraston — DISMISSED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jonathansoufer) [reply README.md:115]: Added

**Review by NEllusion — COMMENTED** [review]
How much of this still fits here vs being moved to https://github.com/MetaMask/metamask-mobile/blob/main/docs/readme/environment.md ?

**Comment by NicolasMassart** [comment]
> How much of this still fits here vs being moved to https://github.com/MetaMask/metamask-mobile/blob/main/docs/readme/environment.md ?

I agree with @NicholasEllul on this, would better be in dev environment setup doc.

**Pull request closed without merging**